crates/amaru/tests/summary.rs

```diff
@@ -123,10 +115,17 @@ fn compare_preprod_snapshot(epoch: u64) {
     .with_unclaimed_refunds(snapshot_from_the_future.as_ref(), &protocol_parameters)
     .unwrap();
 
-    insta::assert_json_snapshot!(format!("rewards_summary_{}", epoch), rewards_summary);
+    insta::with_settings!({
+        snapshot_path => format!("snapshots/{}", network_name)
+    }, {
+        insta::assert_json_snapshot!(
+        format!("rewards_summary_{}", epoch),
+        rewards_summary
+        );
+    });
 }
 
-fn preprod_protocol_version(epoch: Epoch) -> ProtocolVersion {
+fn protocol_version(epoch: Epoch) -> ProtocolVersion {
```

Comment:
The body of this function is only valid for PreProd, though. We might want to pass in the network name here too.